CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to `model-signing` will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+All versions prior to 1.0.0 are untracked.
+
+## [Unreleased]
+
+
+## [1.0.0] - 2024-04-04
+
+### Added
+- First stable release of `model_signing`.
+- Stable, backwards-compatible API.
+- Stable, clean, backwards-compatible CLI wrapping the API.
+- Well-defined signature scheme based on Sigstore bundle and DSSE envelope.
+- Well-defined, future proof, stable, in-toto format for the model signature.
+- Signing and verification using Sigstore, public/private key pairs, and signing certificates.
+- Support for signing/verification via scripts in the repo (`hatch run python -m model_signing`), installing the pip package (`python -m model_signing` or `model_signing`), or Docker containers that wrap around the CLI.
```

Comment:
"installing the pip package" 

Should that be "after installing the pip package"?

Reply:
Oh, I wanted it to be "Support for signing/verification via installing the pip package"